src/providers/LoggerProvider/docs/LoggerProvider.stories.mdx

```diff
@@ -0,0 +1,105 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import { LoggerProvider } from '../';
+
+<Meta title="SDK Providers/LoggerProvider" />
+
+# LoggerProvider
+
+The `LoggerProvider` provides a `Logger` to log component state changes and lifecycle events such as mounting and unmounting.
```

Comment:
Should we just have one guide for both LoggerProvider and useLogger since they are supposed to be used together right?

Reply:
Yeah, we have generally followed a pattern and have documentation for both. But it looks like most of the documentation repeats and is not required. I have deleted the `useLogger` story and have updated the `LoggerProvider` story.